index.js

```diff
@@ -75,11 +75,11 @@ module.exports = {
         new RegExp(path.parse(outputPaths.testSupport.js.testSupport).name + '(.*js)')
       ],
       patterns: [{
-        match: /([^A-Za-z0-9_#]|^|["])define(\W|["]|$)/g,
-        replacement: '$1efineday$2'
+        match: /([^A-Za-z0-9_#]|^|["])define(?=\W|["]|$)/g,
+        replacement: '$1efineday'
```

Comment:
Why remove the `$2` from the replacements?

Reply:
With the lookahead, there is no second grouping, and "$2" will be appended on every match.